### PR TITLE
feat: refactor debug commands

### PR DESCRIPTION
**Pull request by vladdoster**
## Description

Make debug commands easier to use by grouping them under a `debug` command

Screenshot 2023-05-01 at 05 10 03

## Motivation and Context 

- Make it clear that debug commands are related other than they start with `d`
- Clean up the number of base commands

Screenshot 2023-05-01 at 05 07 39

Screenshot 2023-05-01 at 05 07 00

## Usage examples

Screenshot 2023-05-01 at 04 46 21

Screenshot 2023-05-01 at 04 48 27

Screenshot 2023-05-01 at 04 48 13

## Types of changes 

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change
- [X] New feature (non-breaking change which adds functionality)

## Checklist: 

- [ ] All new and existing tests passed.
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: zinit-autoload.zsh [hunk 1]
```diff
@@ -1414,52 +1414,49 @@ EOF
 # User-action entry point.
 .zinit-help() {
            builtin print -r -- "${ZINIT[col-p]}Usage${ZINIT[col-rst]}:
-—— -h|--help|help                – usage information
```
Comment (vladdoster) [inline zinit-autoload.zsh:1417]: Changes here are:

- sorted
- fix `delete` description
- update `debug` command
- remove `d*` commands
- fix `times` command

File: zinit-additional.zsh [hunk 2]
```diff
@@ -101,55 +100,120 @@ function $f {
...
+
+    (( $#o_help )) && {
+        print -l -- $usage
+        return 0
+    }
+
+    cmd="$1"
+    if (( ! $+functions[.zinit-debug-${cmd}] )); then
+        print -l -- $usage
+        return 1
```
Comment (vladdoster) [inline zinit-additional.zsh:213]: Changes:
- logically group the debug sub-commands
- add `status` command
- add conditional log messages

**Comment by vladdoster** [comment]
@psprint, 

Thoughts?

**Comment by psprint** [comment]
I think that it's good idea to group the commands. To clarify, the functions allow to track created variables, functions zstyles and bindkeys.

**Comment by vladdoster** [comment]
@pschmitt,

Can I merge this?

**Comment by pschmitt** [comment]
Oh yeah!

**Review by pschmitt — APPROVED** [review]

**Pull request merged**